config/src/main/java/org/hyperledger/besu/config/FeeMarketConfigOptions.java

```diff
@@ -12,21 +12,16 @@
  *
  * SPDX-License-Identifier: Apache-2.0
  */
-package org.hyperledger.besu.config.experimental;
+package org.hyperledger.besu.config;
 
-import picocli.CommandLine.Option;
+import picocli.CommandLine;
 
-/**
- * Flags defined in this class must be used with caution, and strictly reserved to experimental
- * EIPs.
- */
-public class ExperimentalEIPs {
-  // To make it easier for tests to reset the value to default
-  public static final long EIP1559_BASEFEE_DEFAULT_VALUE = 1000000000L;
+public class FeeMarketConfigOptions {
```

Comment:
This is the only one currently.  I am not sure how much value it is going to provide however, since it is just a configuration for the transition block when the block is not at genesis.  Once mainnet and the major test nets have transitioned, I doubt it will be of much actual use.  That was why I was reluctant to plumb it into the main configuration options and usage.

Reply:
The other experimental eip1559 options like slack coefficient and max change denominator have already been dropped.  Seeing as how we do not have an existing use case for modifying these values *and* geth is specifying these values should be ignored for mainnet:
 https://github.com/ethereum/go-ethereum/blob/master/core/genesis.go#L60-L65
 
 I am just going to remove the initial basefee option and get rid of this class entirely.  If a compelling use case comes along for a 1559-style fee market that uses different constants, we can implement it at that time.  Until then, imo the fewer arcane options the better.

Reply:
👍